packages/apps-v5/src/commands/ps/index.js

```diff
@@ -69,7 +69,21 @@ function * printAccountQuota (context, heroku, app, account) {
   let hours = Math.floor(remainingMinutes / 60)
   let minutes = Math.floor(remainingMinutes % 60)
 
+  let appQuota = quota.apps.find((appQuota) => { return appQuota.app_uuid === app.id })
+  let appQuotaUsed, appPercentage
+  if (appQuota !== undefined) {
+    appQuotaUsed = appQuota.quota_used / 60
+    appPercentage = Math.floor(appQuota.quota_used * 100 / quota.account_quota)
+  } else {
+    appQuotaUsed = 0
+    appPercentage = 0
+  }
```

Comment:
code style is debatable but we can really tighten this up with the following completely untested 🙃 code
```js
const appQuota = quota.apps.find(appQuota => appQuota.app_uuid === app.id)
const appQuotaUsed = appQuota ? appQuota.quota_used / 60 : 0
const appPercentage = appQuota ? Math.floor(appQuota.quota_used * 100 / quota.account_quota) : 0
```

Reply:
Im not a big fan of tighten up the code like that but as you said code style is debatable and given that you and @jdxcode are on board on that I did the commit that fix the styling and also update a similar situation in the same methods a few lines above.

Thanks 🙂